com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -182,39 +182,6 @@ internal bool IsTeleportingNextFrame
             internal float ScaleX, ScaleY, ScaleZ;
             internal double SentTime;
 
-            internal Vector3 Position
```

Comment:
This was removed due to the m_Bitset changes.  Since axial values could potentially be invalid (i.e. 0.0f by default when deserialized), these methods were removed to avoid confusion/errors and the individual float properties are used instead.